src/main/resources/templates/admin.html

```diff
@@ -5,415 +5,416 @@
-													</div>
-													<div class="modal-body team-owner-info-body">
-														<ul>
-															<li><strong th:text="#{acc.info.fullName}"></strong>: <span th:text="${team.owner.firstName + ' ' + team.owner.lastName}"></span></li>
-															<li><strong th:text="#{acc.info.email}"></strong>: <span th:text="${team.owner.email}"></span></li>
-															<li><strong th:text="#{acc.info.phone}"></strong>: <span th:text="${team.owner.phone}"></span></li>
-															<li><strong th:text="#{acc.info.jobTitle}"></strong>: <span th:text="${team.owner.jobTitle}"></span></li>
-															<li><strong th:text="#{acc.info.institution}"></strong>: <span th:text="${team.owner.institution}"></span></li>
-															<li><strong th:text="#{acc.info.website}"></strong>: <span th:text="${team.owner.institutionWeb}"></span></li>
-															<li><strong th:text="#{acc.info.addr1}"></strong>: <span th:text="${team.owner.address1}"></span></li>
-															<li><strong th:text="#{acc.info.addr2}"></strong>: <span th:text="${team.owner.address2}"></span></li>
-															<li><strong th:text="#{acc.info.country}"></strong>: <span th:text="${team.owner.country}"></span></li>
-															<li><strong th:text="#{acc.info.city}"></strong>: <span th:text="${team.owner.city}"></span></li>
-															<li><strong th:text="#{acc.info.region}"></strong>: <span th:text="${team.owner.region}"></span></li>
-															<li><strong th:text="#{acc.info.postalCode}"></strong>: <span th:text="${team.owner.postalCode}"></span></li>
-														</ul>
-													</div>
-												</div>
-											</div>
-										</div>
-									</td>
-									<td th:text="${team.organisationType}"></td>
-									<td th:text="${team.visibility}">Listed Visibility</td>
-									<td>
-										<a href="#" title="Accept" th:href="@{/admin/teams/accept/{teamId}/{teamOwnerId}(teamId = ${team.id}, teamOwnerId = ${team.owner.id})}"><button type="button" class="btn btn-standard admin-click-once-button"><i class="fa fa-check-circle-o fa-lg btn-fa-green"></i> Accept</button></a>
-										<a href="#" title="Reject" th:href="@{/admin/teams/reject/{teamId}/{teamOwnerId}(teamId = ${team.id}, teamOwnerId = ${team.owner.id})}"><button type="button" class="btn btn-standard admin-click-once-button"><i class="fa fa-times-circle-o fa-lg btn-fa-red"></i> Reject</button></a>
-									</td>
-								</tr>
-							</table>
-						</th:block>
-					</div>
+	<body class="mybg loginbackground">
+	<div th:replace="fragments/dashboard_navigation :: dashboardnav">&nbsp;</div>
+	<div class="container">
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+		<!--here make changes-->
+		<!--<div class="tab-content col-sm-10 dashpanel_admin">-->
+		<div class="row dashpanel col-sm-12">
+			<!--<div class="tab-content col-sm-10">-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)